tests/testthat/test-seq_linter.R

```diff
@@ -7,61 +7,62 @@ test_that("other : expressions are fine", {
 
 test_that("finds 1:length(...) expressions", {
   linter <- seq_linter()
+
   expect_lint(
     "function(x) { 1:length(x) }",
-    rex("length(...)", anything, "use seq_len"),
+    rex("length(...)", anything, "Use seq_along"),
     linter
   )
 
   expect_lint(
     "function(x) { 1:nrow(x) }",
-    rex("nrow(...)", anything, "use seq_len"),
+    rex("nrow(...)", anything, "Use seq_len"),
     linter
   )
 
   expect_lint(
     "function(x) { 1:ncol(x) }",
-    rex("ncol(...)", anything, "use seq_len"),
+    rex("ncol(...)", anything, "Use seq_len"),
     linter
   )
 
   expect_lint(
     "function(x) { 1:NROW(x) }",
-    rex("NROW(...)", anything, "use seq_len"),
+    rex("NROW(...)", anything, "Use seq_len"),
     linter
   )
 
   expect_lint(
     "function(x) { 1:NCOL(x) }",
-    rex("NCOL(...)", anything, "use seq_len"),
+    rex("NCOL(...)", anything, "Use seq_len"),
     linter
   )
 
   expect_lint(
     "function(x) { 1:dim(x)[1L] }",
-    rex("dim(...)", anything, "use seq_len"),
+    rex("dim(...)", anything, "Use seq_len"),
     linter
   )
 
   expect_lint(
     "function(x) { 1L:dim(x)[[1]] }",
-    rex("dim(...)", anything, "use seq_len"),
+    rex("dim(...)", anything, "Use seq_len"),
     linter
   )
 })
 
 test_that("1L is also bad", {
   expect_lint(
     "function(x) { 1L:length(x) }",
-    rex("1L:length(...)", anything, "use seq_len"),
+    rex("1L:length(...)", anything, "Use seq_along"),
     seq_linter()
   )
 })
 
 test_that("reverse seq is ok", {
   expect_lint(
     "function(x) { length(x):1 }",
-    rex("length(...):1", anything, "use seq_len"),
+    rex("length(...):1", anything, "Use seq_along"),
```

Comment:
NB this would be `rev(...)`

Reply:
Yea... I think generally the message here can go for better customization. I made _some_ effort there but got stymied by this & gave up for now:

https://github.com/r-lib/xml2/issues/341

Added a TODO for now.